### PR TITLE
broadcast feed

### PR DESCRIPTION
**Pull request by ksamarin90**

### CODE REVIEW

File: src/contract-wrappers/EverscaleMailerV7Wrapper.ts [hunk 1]
```diff
@@ -0,0 +1,712 @@
...
+		includeToMessage: boolean,
+		limit?: number,
+	): Promise<ITVMMessage[]> {
+		return await this.cache.contractOperation(mailer, async (contract, ever, gql, core) => {
+			const dst =
+				subject.type === BlockchainSourceType.DIRECT
+					? subject.recipient
+						? uint256ToAddress(subject.recipient, true, true)
+						: null
+					: subject.feedId;
```
Comment (ksamarin90) [inline src/contract-wrappers/EverscaleMailerV7Wrapper.ts:156]: For broadcast use feedId.

File: src/contract-wrappers/EverscaleMailerV7Wrapper.ts [hunk 2]
```diff
@@ -0,0 +1,712 @@
...
+	formatBroadcastMessage(
+		core: NekotonCore,
+		mailer: ITVMMailerContractLink,
+		message: ITVMInternalMessage,
+	): ITVMMessage {
+		const body = this.decodeBroadcastMessageBody(core, message.body);
+
+		return {
+			isBroadcast: true,
+			feedId: message.dst as Uint256,
```
Comment (ksamarin90) [inline src/contract-wrappers/EverscaleMailerV7Wrapper.ts:121]: Destination should be equal feedId.

File: src/controllers/EverscaleBlockchainController.ts [hunk 3]
```diff
@@ -186,6 +188,10 @@ export class EverscaleBlockchainController extends AbstractBlockchainController
 		// np
 	}
 
+	async getComposedFeedId(feedId: Uint256, count: number) {
```
Comment (ksamarin90) [inline src/controllers/EverscaleBlockchainController.ts:191]: Get feed Id which will be composed while sending broadcast

File: src/controllers/EverscaleWalletController.ts [hunk 4]
```diff
@@ -325,12 +329,6 @@ export class EverscaleWalletController extends AbstractWalletController {
 			return { pushes: [] };
 		} else {
 			const initTime = Math.floor(Date.now() / 1000);
-			const msgId = await this.currentMailer.wrapper.buildHash(
```
Comment (ksamarin90) [inline src/controllers/EverscaleWalletController.ts:328]: Unused.

**Review by ElKornacio — APPROVED** [review]

**Pull request merged**